planning/planning_validator/launch/planning_validator.launch.py

```diff
@@ -38,7 +38,7 @@ def generate_launch_description():
         output="screen",
         remappings=[
             ("~/input/trajectory", LaunchConfiguration("input_trajectory")),
-            ("~/input/kinematics", LaunchConfiguration("odom_topic_name")),
+            ("~/input/kinematics", LaunchConfiguration("odom_topic")),
```

Comment:
Was there a reason to change this?

Reply:
same as https://git.upower.com/group-adas-ad/demo-autonomy/upower_mkz_launch/-/merge_requests/113#note_14794 and https://git.upower.com/group-adas-ad/demo-autonomy/upower_mkz_launch/-/merge_requests/113#note_14796.